docs/configure/triggers.md

```diff
@@ -134,6 +152,39 @@ To configure your trigger by using **JSON** mode instead of **Builder** mode, pa
   ]
 ```
 
+{{% /tab %}}
+{{% tab name="JSON Template: Conditional Data Ingestion" %}}
+
+```json {class="line-numbers linkable-line-numbers"}
+"triggers": [
+    {
+      "name": "<trigger name>",
+      "event": {
+        "type": "conditional_data_ingested",
+        "conditional": {
+          "data_capture_method": "<component>:<name-of-component>:<method>",
+          "condition": {
+            "evals": [
+              {
+                "operator": "<operator>",
```

Comment:
Please list the operators, so readers don't have to guess (I know they're further down but this makes it easier):
```suggestion
                "operator": "<lt|gt|lte|gte|eq|neq>",
```